src/test/ui/print_type_sizes/padding.stdout

```diff
@@ -0,0 +1,21 @@
+print-type-size type: `E1` overall bytes: 24 align: 8
+print-type-size    discrimimant bytes: 4
+print-type-size    variant A exact bytes: 16
+print-type-size        padding bytes: 4
+print-type-size        field .0 bytes: 8 align: 8
```

Comment:
This code in principle can print the offset assigned to each field (its being passed back).

However my own experiments indicate that the computed offsets from the `layout` module do not always match what we use in the representation, so I decided not to print out the offsets in this version of the code.

Whenever we get around to rearranging the field order, we will *have* to fix the `layout` module to actually compute/maintain correct offset values, and we can revise this accordingly then.